include/zephyr/bluetooth/hci.h

```diff
@@ -1278,6 +1278,9 @@ struct bt_hci_cp_le_set_adv_set_random_addr {
 #define BT_HCI_LE_ADV_PROP_ANON                 BIT(5)
 #define BT_HCI_LE_ADV_PROP_TX_POWER             BIT(6)
 
+#define BT_HCI_LE_PRIM_ADV_INTERVAL_MIN         0x000020
+#define BT_HCI_LE_PRIM_ADV_INTERVAL_MAX         0xFFFFFF
```

Comment:
Should these contain `EXT` to explicitly verify that these are for extended advertising? 
Legacy primary adv interval is still maximum 0x4000:
![image](https://user-images.githubusercontent.com/2520413/165941168-4b189202-7b11-40a6-810f-c6ca442c532d.png)

Reply:
Legacy advertising interval are defined here:
https://github.com/zephyrproject-rtos/zephyr/blob/672835730b4ae870a1b92d115342444642f87eb6/include/zephyr/bluetooth/hci.h#L842-L844

Reply:
Right, but to the untrained eye these may be confusing to some, but I guess the location of the `#define` is a good enough hint that they are for extended advertising :)